### ISSUE TITLE
Use of subsites pages without a banner causing accessibility issue with page title

### ISSUE THREAD

**Issue by cjstevens78** [localgovdrupal/localgov_subsites#26]
The ```<h1>``` title for subsites pages is contained within the banner area. When a page is published without a banner it fails accessibility checks due to not having an ```<h1>``` title. @willguv suggested that the intention for subsites is to always have a banner as part of the page structure. If this is the case then there should be some kind of validation in place to prevent a page being published without one. See screenshot of a page on Croydon that was published without a banner. Screen readers are looking for a page title ```<h1>``` but instead find a section title ```<h2>```.

<img width="1371" alt="Screenshot 2021-03-29 at 13 44 39" src="https://user-images.githubusercontent.com/11304107/112992645-424a8500-9160-11eb-839f-8e80d2058297.png">

**Comment by andybroomfield** [comment]
Just to add, it's not always the case a subsite will have a banner.
eg. https://www.brighton-hove.gov.uk/how-stay-active-home
The missing `<h1>` is down to the subsites banner changing to a paragraph, so the block does not get rendered if no banner is set, so it will not render the page title. 

I think the best cause of action is to provide a default `<h1>` with the title if the banner is not set, rather than assume it will always be present and make it a required field, as we don't know if every subsite will contain a banner image.

**Comment by keelanfh** [comment]
@msayoung

**Comment by bisd** [comment]
I am trying to look into this. Is this the kind of page we are talking about?

https://test.localgovdrupal.org/test-subsite-demo-all-components/test-pages-documents

**Comment by markconroy** [comment]
I'm going to close this issue in favour of https://github.com/localgovdrupal/localgov_subsites/issues/116
